x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/utils.js

```diff
@@ -149,7 +149,11 @@ function showResults(resp, action) {
 export function cloneJob(jobId) {
   loadFullJob(jobId)
     .then((job) => {
-      mlJobService.currentJob = job;
+      if(job.custom_settings && job.custom_settings.created_by) {
```

Comment:
Can you maybe explain in a comment why these two attributes are used to determine whether a config is put into `tempJobCloningObjects`?